src/bidiMapper/modules/context/BrowsingContextProcessor.ts

```diff
@@ -130,6 +131,15 @@ export class BrowsingContextProcessor {
       throw err;
     }
 
+    if (params.background !== true) {
```

Comment:
I believe Target.createTarget with background == false should already activate the new target, is it not the case?

Reply:
agree, removed